src/testRunner/unittests/moduleResolution.ts

```diff
@@ -588,23 +588,23 @@ export = C;
                 "/a/b/c.ts": `/// <reference path="D.ts"/>`,
                 "/a/b/d.ts": "var x"
             });
-            test(files, { module: ModuleKind.AMD, forceConsistentCasingInFileNames: true }, "/a/b", /*useCaseSensitiveFileNames*/ false, ["c.ts", "d.ts"], [1149]);

```

Comment:
We should have test cases when `forceConsistentCasingInFileNames` is set to false now that its not default.